packages/interbit-ui-components/src/scss/interbit/_activity-bar.scss

```diff
@@ -0,0 +1,69 @@
+.ibweb-activity-bar {
+  min-width: 320px;
+  max-width: 600px;
+  color: #000;
```

Comment:
Please use the colour variables defined in `src/scss/interbit/_variables.scss` rather than hard coding them.